packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -89,6 +88,35 @@ export type OdspError =
     | DriverError
     | IOdspError;
 
+export function getSPOAndGraphRequestIdsFromResponse(headers: { get: (id: string) => string | undefined | null}) {
+    interface LoggingHeader {
+        headerName: string;
+        logName: string;
+    }
+    // We rename headers so that otel doesn't scrub them away. Otel doesn't allow
+    // certain characters in headers including '-'
+    const headersToLog: LoggingHeader[] = [
```

Comment:
A naive question perhaps but why do we change the string between the headerName and logName? The comment above explains remove characters like '-' but why do we also change the string contents, i.e. X-Fluid-Retries -> serverRetries? Is this to preserve the values uniformly from historical telemetry to future telemetry?

Reply:
I do not know - it was before me :) I'd say that I like "requestId" and if I did not from scratch, I'd probably do the same thing for sprequestguid. But given it was there (on most of the paths) in lower case, I'm consolidating on that name to reduce amount of changes. I'd say across all of them, I like the least xMsedgeRef - hard to read and I do not know why we need it (it was collected on one of the calls).
I'd say - we should continue to improve here. It might be good idea to take a pass over all of our telemetry properties, define standard and go after uniform approach. Leaving as is in this PR, that's a bit out of scope here.